src/scmdata/run.py

```diff
@@ -287,9 +293,17 @@ def _from_ts(
     return df, meta
 
 
-class ScmRun:  # pylint: disable=too-many-public-methods
+class BaseScmRun:  # pylint: disable=too-many-public-methods
     """
-    Data container for holding one or many time-series of SCM data.
+    Base class of a data container for timeseries data
+    """
+
+    required_cols = ("variable", "unit")
+    """
+    Required metadata columns
+
+    This is the bare minimum columns which are expected. Attempting to create a run
+    without the metadata columns specified by :attr:`required_cols` will raise a ValueError
```

Comment:
```suggestion
    without the metadata columns specified by :attr:`required_cols` will raise a MissingRequiredColumn
```

We should probably rename `MissingRequiredColumn` to `MissingRequiredColumnError`